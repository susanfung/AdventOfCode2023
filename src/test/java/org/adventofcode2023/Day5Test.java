package org.adventofcode2023;

import org.approvaltests.Approvals;
import org.junit.jupiter.api.Test;

import java.util.Arrays;
import java.util.List;

class Day5Test {
    @Test
    void createMap() {
        List<String> seedToSoilMap = Arrays.asList("50 98 2", "52 50 48");
        Approvals.verify(Day5.createMap(seedToSoilMap));
    }
}