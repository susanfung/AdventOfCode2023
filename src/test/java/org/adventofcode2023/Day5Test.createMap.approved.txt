50 : 52 
51 : 53 
52 : 54 
53 : 55 
54 : 56 
55 : 57 
56 : 58 
57 : 59 
58 : 60 
59 : 61 
60 : 62 
61 : 63 
62 : 64 
63 : 65 
64 : 66 
65 : 67 
66 : 68 
67 : 69 
68 : 70 
69 : 71 
70 : 72 
71 : 73 
72 : 74 
73 : 75 
74 : 76 
75 : 77 
76 : 78 
77 : 79 
78 : 80 
79 : 81 
80 : 82 
81 : 83 
82 : 84 
83 : 85 
84 : 86 
85 : 87 
86 : 88 
87 : 89 
88 : 90 
89 : 91 
90 : 92 
91 : 93 
92 : 94 
93 : 95 
94 : 96 
95 : 97 
96 : 98 
97 : 99 
98 : 50 
99 : 51 
